### ISSUE TITLE
B2C password reset results in “password has expired” error

### ISSUE THREAD

**Issue by johndowns** [johndowns/soscafe#94]
Whenever we reset a user’s password using the B2C UI, we can reset the password to a temporary password provided by B2C - but then when the user logs in they get an error saying “Password has expired”, and there seems to be no way for them to log in using that temporary password.

@chrispadgettlivecom any ideas? :)

**Comment by dwdowns** [comment]
Any update on this? One cafe owner is unable to get in (Manuka cafe)

**Comment by johndowns** [comment]
Have they tried using the ‘I forgot my password’ option? What happens?

**Comment by dwdowns** [comment]
Yes, and when they get in they can't see any entries. I wanted to try to log in as them to see why.  I've bee back and forth and seen screen shots; can't work out why

**Comment by johndowns** [comment]
Can you email me (not on here) the email address and vendor ID?

**Comment by dwdowns** [comment]
done

**Comment by dwdowns** [comment]
FYI the urgency has gone out of this as he managed to log in. Thanks
